configuration/globalproperties/activePatientsByLocation.xml

```diff
@@ -0,0 +1,34 @@
+<config>
+  <globalProperties>
+    <globalProperty>
+      <property>emrapi.sqlSearch.activePatientsByLocation</property>
+      <value>
+      select distinct concat(pn.given_name," ", ifnull(pn.family_name,'')) as name,
+      pi.identifier as identifier,
+      concat("",p.uuid) as uuid,
+      concat("",v.uuid) as activeVisitUuid,
+      IF(va.value_reference = "Admitted", "true", "false") as hasBeenAdmitted
+      from
+      visit v join person_name pn on v.patient_id = pn.person_id and pn.voided = 0 and v.voided=0
+      join patient_identifier pi on v.patient_id = pi.patient_id and pi.voided=0
+      join patient_identifier_type pit on pi.identifier_type = pit.patient_identifier_type_id
+      join global_property gp on gp.property="bahmni.primaryIdentifierType" and gp.property_value=pit.uuid
+      join person p on p.person_id = v.patient_id  and p.voided=0
+      join encounter en on en.visit_id = v.visit_id and en.voided=0
+      join visit_type vt on vt.visit_type_id=v.visit_type_id 
+      left outer join location loc on en.location_id = loc.location_id
+      join encounter_provider ep on ep.encounter_id = en.encounter_id  and ep.voided=0
+      join provider pr on ep.provider_id=pr.provider_id and pr.retired=0
+      join person per on pr.person_id=per.person_id and per.voided=0
```

Comment:
@prashantsable1991 this JOIN looks similar to line 16, could you please double check?

Reply:
> @prashantsable1991 this JOIN looks similar to line 16, could you please double check?

If you check on CMR-DEV you will find on which line I have made changes else I kept as it is.